static/js/settings_users.js

```diff
@@ -428,19 +428,60 @@ function get_human_profile_data(fields_user_pills) {
 }
 
 export function confirm_deactivation(user_id, handle_confirm, loading_spinner) {
+    const bots_owned_by_user = bot_data.get_all_bots_owned_by_user(user_id);
+    let bot_text;
+    if (bots_owned_by_user.length === 0) {
+        bot_text = $t_html({defaultMessage: "does not administer any active bots."});
+    } else {
+        bot_text = $t_html({defaultMessage: "administers the following active bots:"});
```

Comment:
This internationalization approach doesn't work -- different languages have different word orders, so one needs to always pass entire sentences to the i18n tools.

Likely we need to move this logic into the template, and just have a 2x2=4 variants of the overall string, if we're trying to explain all these things as a compound sentence (which I generally don't recommend because of the exploding strings that this sort of compound sentence requires). See https://zulip.readthedocs.io/en/latest/translating/internationalization.html